File: python/standardization/basic/src/Algorithm.py
from Devtools import AlgorithmHandler
import Algorithmia

# API calls will begin at the apply() method, with the request body passed as 'input'
# For more details, see algorithmia.com/developers/algorithm-development/languages
def foo(input, context = None):
    return "hello {}".format(input)

def configure():
    algorithm = AlgorithmHandler(foo)
    return algorithm

File: languages/r34/template/src/Algorithm.r
library(algorithmia)
# API calls will begin at the algorithm() method, with the request body passed as 'input'
# For more details, see algorithmia.com/developers/algorithm-development/languages

algorithm <- function(input) {
  paste("hello", input)
}

load <- function() {
  # Defines the loading function if defined
}

algo <- getAlgorithmHandler(algorithm, load)
algo$run()

